src/tokens/fontSize.js

```diff
@@ -1,6 +1,8 @@
 const light = (core) => ({
   // Globals
   ...core.FontSize,
+  // Breadcrumbs
+  Breadcrumbs: core.FontSize.Body,
```

Comment:
Do we need separate Font and FontSize tokens for this, or can you just use the `Text` component?

Reply:
Are we trying to move towards a pattern of more granularity with tokens, or more towards components depending on each other? Is it something thats been discussed already? If not, my 2 cents is I would personally lean towards token granularity. Components relying on each other could get messy down the line and a headache to work with especially in getting versions to all align, instead of just updating the design tokens.